forc-util/src/lib.rs

```diff
@@ -82,22 +102,22 @@ pub fn find_parent_dir_with_file(starter_path: &Path, file_name: &str) -> Option
     None
 }
 /// Continually go up in the file tree until a Forc manifest file is found.
-pub fn find_manifest_dir(starter_path: &Path) -> Option<PathBuf> {
+pub fn find_manifest_dir_in_parent(starter_path: &Path) -> Option<PathBuf> {
```

Comment:
```suggestion
pub fn find_manifest_dir_in_parents(starter_path: &Path) -> Option<PathBuf> {
```
Maybe we should use plural here to as the manifest might not be in the immediate parent, but some grandparent.

Reply:
Thinking further, maybe a simpler name would be:
```suggestion
pub fn find_parent_manifest_dir(starter_path: &Path) -> Option<PathBuf> {
```
This might be a little more consistent with the `find_parent_dir` and `find_nested_manifest_dir` function names? Maybe we should change the `with_check` function too for consistency? I'm picturing:
- `find_parent_dir_with_file`
- `find_nested_dir_with_file`
- `find_parent_manifest_dir`
- `find_nested_manifest_dir`
- `find_parent_manifest_dir_with_check`

Reply:
Sounds great!